src/core/scene/a-scene.js

```diff
@@ -313,19 +313,29 @@ module.exports = registerElement('a-scene', {
         }
 
         this.addEventListener('loaded', function () {
-          if (this.renderStarted) { return; }
+          AEntity.prototype.play.call(this);  // .play() *before* render.
```

Comment:
I tried to get it working by calling `play` only after camera, but couldn't get the tests passing. I might have some code quality improvements though.